packages/api/src/hooks/useTradingAccountsList.ts

```diff
@@ -13,7 +13,11 @@ const useTradingAccountsList = () => {
 
     // Add additional information to each trading account.
     const modified_accounts = useMemo(() => {
-        return filtered_accounts?.map(trading => ({ ...trading }));
+        return filtered_accounts?.map(trading => ({
+            ...trading,
+            first_real_account: filtered_accounts?.find(account => account.account_type === 'real')?.loginid[0],
```

Comment:
we should get the first item of filtered array, right?
```suggestion
            first_real_account: filtered_accounts?.find(account => account.account_type === 'real')[0]?.loginid,
```

Reply:
Yes. I think it's necessary to have the `[0]`

Reply:
So  it should be `(account => account.account_type === 'real')[0]?.loginid` instead of `find(account => account.account_type === 'real')?.loginid[0],`